tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/ImdsV2Tests.cs

```diff
@@ -1462,5 +1462,42 @@ private static void AssertCertSubjectCnDc(X509Certificate2 cert, string expected
         }
 
         #endregion
+        
+        #region Fallback Behavior Tests
+        // Verifies non-mTLS request after IMDSv2 detection falls back per-request to IMDSv1 (Bearer),
+        [DataTestMethod]
+        [DataRow(UserAssignedIdentityId.None, null)]
+        [DataRow(UserAssignedIdentityId.ClientId, TestConstants.ClientId)]
```

Comment:
To be consistent with all other test, can you also add cases for ResourceId and ObjectId?